fullstop/src/main/resources/db/migration/V6__create_outdated_ami_type.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO fullstop_data.violation_type (
```

Comment:
I used the stored procedure, but I would like to add the issue number in the audit columns.
I can refactor the store procedure to achieve that.

I will move the db/migration folder ;)